pages/mesosphere/dcos/services/include/forwarding-dns-cusotm-domain.tmpl

```diff
@@ -0,0 +1,15 @@
+Every DC/OS cluster has a unique cryptographic ID which can be used to forward DNS queries to that cluster. To securely expose the service outside the cluster, external clients must have an upstream resolver configured to forward DNS queries to the DC/OS cluster of the service as described [here](/mesosphere/dcos/latest/networking/DNS/mesos-dns/expose-mesos-zone/).
```

Comment:
Can we correct the typo with the file name?
it should be `forwarding-dns-custom-domain.tmpl`

Reply:
good catch!